server/src/entities/classes/destroyables/movables/characters/mobs/LargeAdumbral.js

```diff
@@ -32,6 +32,17 @@ class LargeAdumbral extends Mob {
     }
 
     onAllHitPointsLost() {
+        const { dayPhase } = this.board;
+
+        if (dayPhase === DayPhases.Day) {
+            // Give 33% of full glory value if it's daytime
+            this.gloryValue = `${+this.gloryValue * 0.33}`;
```

Comment:
Why the use of template literals here?

Reply:
To convert `gloryValue` back to a string after multiplication, which I assume is it's original type based on the unary operator `+` being used in  [`onAllHitPointsLost()`](https://github.com/Arcanorum/dungeonz/blob/master/server/src/entities/classes/destroyables/movables/characters/mobs/Mob.js#L82)

Reply:
Humm... now that you mention it IDK why that `+` is there in that method. Looks like it can be removed.

The `gloryValue` should always be a number though.